gemfiles/rails_6.0.gemfile

```diff
@@ -4,5 +4,6 @@ source "https://rubygems.org"
 
 gem "capybara", "~> 3"
 gem "rails", "~> 6.0.0"
+gem "tailwindcss-rails", "~> 2.0"
```

Comment:
I think 2 as a dependency is what we are looking for, since `tailwindcss-rails` version 1 was not complete and it was missing some of the major tailwindcss parts. That is solved with version 2, so I can't really see anyone using version 1 :)